app/src/main/java/com/dgaf/happyhour/Model/Adapter/RestaurantAdapter.java

```diff
@@ -98,16 +109,35 @@ public boolean onTouch(View v, MotionEvent event) {
         }
     }
 
-    public RestaurantAdapter(Fragment fragment, String restaurantId, View fragmentView) {
+    public RestaurantAdapter(Fragment fragment,View fragmentView, String restaurantId, String dealId) {
         this.fragment = fragment;
         this.imageLoader = ImageLoader.getInstance();
         this.expandDealListAdapter = new ExpandDealListAdapter(fragment.getActivity(), restaurantId);
         this.restaurant = new RestaurantModel();
+        this.deal = new DealModel();
+        parseLocation = getLocation();
         createViewHolders(fragmentView);
-        loadRestaurantDetails(restaurantId);
+        loadRestaurantDetails(restaurantId, dealId);
     }
 
-    public void loadRestaurantDetails(String restaurantId) {
+    public ParseGeoPoint getLocation() {
+        // Geisel Library - Default Location
+        double latitude = 32.881122;
+        double longitude = -117.237631;
+        if (!Build.FINGERPRINT.startsWith("generic")) {
```

Comment:
We probably want to put Geisel's location as a constant at the top rather than just place the coordinates as a local var here.  For code style's sake.